feature/policy_forwarding/encapsulation/otg_tests/staticgueencap_and_bgp_path_selection/README.md

```diff
@@ -0,0 +1,382 @@
+       * IPv6Prefix8/64 to IPv6Prefix3/64 [DSCP:AF2]
+       * IPv4Prefix9/24 to IPv4Prefix4/24 [DSCP:AF3]
+       * IPv6Prefix9/64 to IPv6Prefix4/64 [DSCP:AF3]
+       * IPv4Prefix10/24 to IPv4Prefix5/24 [DSCP:AF4]
+       * IPv6Prefix10/64 to IPv6Prefix5/64 [DSCP:AF4]
+
+   * Following flows from ATE2:Port1 to ATE1:Port1 are sent Unencaped. 
+       * IPv4Prefix6/24 to IPv4Prefix1/24 [DSCP:BE1]
+       * IPv6Prefix6/64 to IPv6Prefix1/64 [DSCP:BE1]
+       * IPv4Prefix7/24 to IPv4Prefix2/24 [DSCP:AF1]
+       * IPv6Prefix7/64 to IPv6Prefix2/64 [DSCP:AF1]
+       * IPv4Prefix8/24 to IPv4Prefix3/24 [DSCP:AF2]
+       * IPv6Prefix8/64 to IPv6Prefix3/64 [DSCP:AF2]
+       * IPv4Prefix9/24 to IPv4Prefix4/24 [DSCP:AF3]
+       * IPv6Prefix9/64 to IPv6Prefix4/64 [DSCP:AF3]
+       * IPv4Prefix10/24 to IPv4Prefix5/24 [DSCP:AF4]
+       * IPv6Prefix10/64 to IPv6Prefix5/64 [DSCP:AF4]
+<br><br><br>
+
+### Following Health checks to be run Pre and Post every sub test
+  * No system/kernel/process/component coredumps
+  * No high CPU spike or usage on control or forwarding plane
+  * No high memory utilization or usage on control or forwarding plane
+  * No processes/daemons high CPU/Memory utilization
+  * No generic drop counters
+    * QUEUE drops
+    * Interfaces
+    * VOQ
+    * Fabric drops
+    * ASIC drops
+  * No flow control frames tx/rx
+  * No CRC or Layer 1 errors on interfaces
+  * No config commit errors
+  * No system level alarms
+  * In spec hardware should be in proper state
+    * No hardware errors
+    * Major Alarms
+  * No HW component or SW processes crash
+Openconfig-paths for the above are covered in the `OpenConfig Path and RPC Coverage` section below<br><br><br>
+
+
+**PF-1.6.1: [Baseline]**
+   * Situation:
+       * BE1-AF4 traffic from ATE1:Port1 to ATE2:Port1 while there are no IBGP advertisements from ATE2:Port2 to DUT:Port3
+       * Unencaped traffic received from ATE2 to ATE1:Port1 is routed via the IBGP session between DUT:Port2 and ATE2:Port1<br>
+
+   * Test Steps:
+       * Please run all the health checks suggested above prior to starting the test as a baseline.
+       * Configure DUT as suggested above.
+           * Make sure the IBGP peering between DUT:Port3 and ATE2:Port2 are down for this subtest.
+       * Start above mentioned flows
+           * Dont start the above mentioned encaped flows from ATE2:Port3<br>
+           
+   * Expectations:
+	    * ATE1:Port1 to ATE2:Port1 flows are expected to be 100% successful and routing over the IBGP peering between DUT:Port2 and ATE2:Port1
+	    * ATE2 to ATE1:Port1 Unencaped flows must be 100% successful. The flows are expected to route through the IBGP peering between ATE2:Port1 --> DUT:Port2 --> ATE
+	    * Run post test Healthchecks and compare. Ensure there are no drops, coredumps etc.<br><br><br>
+
+
+**PF-1.6.2: Migrate routing of BE1 flows from ATE1:Port1 to ATE2:Port1 to ATE2:Port3 instead**
+   * Situation:
```

Comment:
In this scenario -- I think it has become too complex defining the prefixes to selectively advertise in each test case. I'd suggest that we break this down in terms of the prefixes that we have in all scenarios, and then the deltas that are being created per test case.

The alternative is to define these prefixes in terms of sets (and the sessions with a name) and say, for example:

> ### Definitions:
> * `FLOW-SET-1`: ATE port 1 to ATE port 2
> * `BGP-SESSION-1`; ATE port 1 to ATE port 2, iBGP with AS100, policy setting local pref to 42 for all routes.
>
> ### Test Case
> Enable FLOW-SET-1 and BGP-SESSION-1.

I think also defining this in terms of the positive (i.e., enable flow 1) rather than the negative (i.e., disable flow 2) is clearer, because the reader can then just understand that they enable the list of flows that you specified. I can imagine writing this like:

```golang

flows := map[string]*someFlowThingy{
  "flow1": {
     src: "ATE-1",
     dst: "ATE-2",
     // some spec of the flow
  },
  "flow2": {
    src: "ATE-3",
    dst: "ATE-4",
  },  
}

func TestSomething(t *testing.T) {
   enableFlows(t, "flow1", "flow2", ...)
}
```

which is pretty easy to read.